js/bigone.js

```diff
@@ -1228,6 +1240,84 @@ module.exports = class bigone extends Exchange {
         return this.parseTransactions (withdrawals, code, since, limit);
     }
 
+    async transfer (code, amount, fromAccount, toAccount, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const accountsById = this.safeValue (this.options, 'accountsByType', {});
+        const fromId = this.safeString (accountsById, fromAccount);
+        if (fromId === undefined) {
+            const keys = Object.keys (accountsById);
+            throw new ExchangeError (this.id + ' fromAccount must be one of ' + keys.join (', '));
+        }
+        const toId = this.safeString (accountsById, toAccount);
+        if (toId === undefined) {
+            const keys = Object.keys (accountsById);
+            throw new ExchangeError (this.id + ' toAccount must be one of ' + keys.join (', '));
+        }
+        const type = this.safeString (params, 'type');
+        const subAccount = this.safeString (params, 'sub_account');
+        const guid = this.safeString (params, 'guid', this.uuid ());
+        const request = {
+            'symbol': currency['id'],
+            'amount': this.currencyToPrecision (amount),
```

Comment:
Missed the currency code argument here:
```JavaScript
            'amount': this.currencyToPrecision (amount),
```
↓
```JavaScript
            'amount': this.currencyToPrecision (code, amount),
```

Reply:
thanks!